### PR TITLE
Adding a command to add VM to cleanroom recovery group and changing the integration name

### PR DESCRIPTION
**Pull request by Cv-securityIQ**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by samuelFain** [comment]
Hi @Cv-securityIQ,
Thank you for your contribution!
Your PR review is slightly delayed and we will review it as soon as possible, we apologize for the inconvenience. 
In the meantime, please make sure the registration form is filled.
I'm available for any of your questions here or on Slack.
Thanks again.

**Comment by samuelFain** [comment]
Before we proceed with the merge, there are a couple of final steps to address:

- Kindly apply the `Partner-Approved` label to the pull request. This signifies that you, as the partner, acknowledge and accept the updates made in this PR to your pack.
- To verify that the implemented changes are operating correctly, a demo is required. I suggest recording a brief video showcasing the execution of `commvault-security-set-cleanroom-add-vm-to-recovery-group` with the expected behavior and share it with me. alternatively, we can schedule a brief meeting for a live demo.

Once these items are checked off, we will proceed with the merge. Thank you for your cooperation and patience.

File: Packs/CommvaultSecurityIQ/Integrations/CommvaultSecurityIQ/CommvaultSecurityIQ.py [hunk 1]
```diff
@@ -350,6 +351,7 @@ def __init__(self, base_url: str, verify: bool, proxy: bool):
         Constructor to initialize the Commvault client object
         """
         super().__init__(base_url=base_url, verify=verify, proxy=proxy)
+        # print('Initialing base url as [{}]'.format(base_url))
```
Comment (samuelFain) [inline Packs/CommvaultSecurityIQ/Integrations/CommvaultSecurityIQ/CommvaultSecurityIQ.py:354]: Please remove commented code throughout this file.

**Comment by Cv-securityIQ** [comment]
@samuelFain Can you please elaborate on what needs to be done for **Partner-approved** label.

**Comment by Cv-securityIQ** [comment]
@samuelFain , attached the video showing the execution of the command `commvault-security-set-cleanroom-add-vm-to-recovery-group`. Also, I don't see the edit option where you edit the label on a PR. Can you please let us know how we can add it?

**Comment by samuelFain** [comment]
@Cv-securityIQ Thanks for the demo screen recording!
Also, the `Partner-approved` label has been added.
I will now proceed to merge the PR.

Thanks again!

**Pull request merged**